spec/commands/viz_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe "bundle viz" do
+  before :each do
+    `gem install ruby-graphviz`
+  end
+
+  context "with a standard Gemfile" do
+    before :each do
+      `gem install httpi -v "= 2.2.7"` # requires any rack
```

Comment:
I'm unfortunately not familiar enough with the Bundler test suite to know how to do that. :( I mentioned that in https://github.com/bundler/bundler/pull/3629#discussion-diff-29895133.